performance/Performance.StackExchange.Redis/Properties/launchSettings.json

```diff
@@ -0,0 +1,19 @@
+{
+  "profiles": {
+    "Performance.StackExchange.Redis": {
+      "commandName": "Project",
+      "environmentVariables": {
+        "COR_ENABLE_PROFILING": "1",
+        "COR_PROFILER": "{846F5F1C-F9AE-4B07-969E-05C26BC060D8}",
```

Comment:
How did the profiler work without `integrations.json` or the `COR*_PROFILER_PATH` variables?

Reply:
Convention, new from Zach. The assumption is that everything we need exists in this directory.

Reply:
> new from Zach

Can you point me to this change? Was it communicated to the team?

I can see convention working with `integrations.json`, but `COR*_PROFILER_PATH` is required by .NET (unless you're regestring the COM components).

Reply:
Hmmm, right, maybe I had leftover env variables from my MSI, let me check.

Reply:
I was testing with the msi install at first and then disabled the tracer to look for differences but all the needed variables were there still and the version matched up.